apps/ui/src/components/TransactionsListItem.vue

```diff
@@ -22,7 +22,11 @@ const title = computed(() => {
   }
 
   if (props.tx._type === 'contractCall') {
-    return `Contract call to <b>_NAME_</b>`;
+    return 'Contract call to <b>_NAME_</b>';
+  }
+
+  if (props.tx._type === 'raw') {
+    return 'Raw transaction to <b>_NAME_</b>';
```

Comment:
Isn't this the same as a contract call? Maybe we can merge these instead of creating a new tx type

Reply:
Those don’t have ABI/method/params, we only have raw details (data, value, contract).

We can’t derive those fields that ContractCall transaction requires so i created new type for it.